src/erlcloud_s3.erl

```diff
@@ -34,6 +36,10 @@
          get_bucket_and_key/1
         ]).
 
+-ifdef(TEST).
```

Comment:
Its exported so we can test it within the framework of the existing tests. Personally i'd prefer to have the eunit tests in the same file, but thats not how the rest of the library does it.